comitup/iwscan.py

```diff
@@ -83,7 +84,7 @@ def devaps(dev: str, dump: str = "") -> List[Dict[str, str]]:
             ap = blk2dict(blk)
             ap["power"] = dbm2pct(float(ap["signal"].split()[0]))
 
-            if ap["SSID"] and ap["SSID"].count(r"\x00") == 0:
+            if ap["SSID"] and ( ap["SSID"].count(r"\x00") == 0 or ap["SSID"].count(r"'") == 0 ):
```

Comment:
This part doesn't look right. It is supposed to hide APs that are supposed to be hidden (suggesting that the terms should be AND'd). Does your fix work without this change?